extensions/sql-migration/src/constants/strings.ts

```diff
@@ -483,6 +489,7 @@ export const ISSUES_DETAILS = localize('sql.migration.issues.details', "Issue de
 export const SELECT_DB_PROMPT = localize('sql.migration.select.prompt', "Click on SQL Server instance or any of the databases on the left to view its details.");
 export const NO_ISSUES_FOUND_VM = localize('sql.migration.no.issues.vm', "No issues found for migrating to SQL Server on Azure Virtual Machine.");
 export const NO_ISSUES_FOUND_MI = localize('sql.migration.no.issues.mi', "No issues found for migrating to SQL Server on Azure SQL Managed Instance.");
+export const MEMENTO_STRING = localize('sql.migration.memento.string', "sqlMigration.assessmentResults");
```

Comment:
Is this possibly a typo or should this be a constant instead of being localized?  will 'sqlMigraiton.assessmentResults' change with a different language

Reply:
Yes, memento IDs should never be localized.